### DISCUSSION TITLE
Yansongda\Pay\decrypt_wechat_resource_aes_256_gcm(): Return value must be of type array|string, bool returned

### DISCUSSION THREAD

**Discussion by kkkreset** [yansongda/pay#907] [Q&A]
PHP 8.2.3
Laravel 10.38.1
 \Yansongda\Pay\Pay::config(config('pay'));

        $params = [

            '_config' => 'default' // 多租户配置时使用

        ];

        \Yansongda\Pay\get_wechat_public_certs($params);

**Comment by yansongda** [comment]
检查微信v3秘钥
